frontend/viewer/src/lib/components/field-editors/field-title.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import {t} from 'svelte-i18n-lingui';
+  import FieldHelpIcon from '../../entry-editor/FieldHelpIcon.svelte';
+  import {useCurrentView} from '$lib/views/view-service';
+
+  const {
+    liteName,
+    classicName,
+    helpId,
+  }: {
+    liteName: string;
+    classicName: string;
+    helpId: string | undefined;
+  } = $props();
+
+  const view = useCurrentView();
+
+  const { label, title } = $derived(
+    $view.type === 'fw-classic'
+      ? {
+          label: classicName,
+          title: $t`${liteName} (FieldWorks Lite)`,
+        }
+      : {
+          label: liteName,
+          title: $t`${classicName} (FieldWorks)`,
+        },
+  );
+
+  // kind of crazy, but I don't think Svelte's white-space handling let's us use &nbsp; between the label and help icon
```

Comment:
not sure I understand this comment or how it relates. If you want to insert a non breaking space you could just do this `const nonBreakingSpace = '\u00A0';` and in the html `{nonBreakingSpace}` and svelte won't remove that.

Reply:
I believe `&nbsp;` is the "better" way to use `\u00A0` in HTML.
The problem is that there's no way to make `&nbsp;` the **only** space character between the label and the inline icon. The fact that the icon is a seperate component and that there's an `if` involved etc. means there's whitespace that's effectively outsided our control.

Reply:
Actually, I sort of pinned it down to the fact that icons are `display: inline-block`, which seems to make `&nbsp;` not work.
So, I still don't know how else to do it except the way I have it.